contracts/Splitter.sol

```diff
@@ -33,4 +33,12 @@ contract Splitter is Pausable, Killable {
     emit LogWithdrawn(msg.sender, balanceToWithdraw);
     msg.sender.transfer(balanceToWithdraw);
   }
+
+  function withdrawAll() external onlyOwner whenKilled {
+    uint256 balanceToWithdraw = address(this).balance;
+    require(balanceToWithdraw > 0, "Balance must be grater than zero to withdraw!");
```

Comment:
I think this line of code becomes irrelevant because the normal withdraw is locked now.